src/providers/ogr/qgsogrprovider.cpp

```diff
@@ -1524,6 +1524,8 @@ bool QgsOgrProvider::renameAttributes( const QgsFieldNameMap& renamedAttributes
 
 bool QgsOgrProvider::changeAttributeValues( const QgsChangedAttributesMap &attr_map )
 {
+  bool hasTransaction;
```

Comment:
Can you move this declaration down to line 1539 when the value is assigned? Then we don't have an uninitisalised variable floating around.